demos/projects/ST/b-l475e-iot01a/config/demo_config.h

```diff
@@ -506,12 +506,18 @@ static unsigned char root_cert_array[] = {
  *  WIFI_ECN_WPA2_PSK = 0x03,
  *  WIFI_ECN_WPA_WPA2_PSK = 0x04
  */
-#define WIFI_SECURITY_TYPE               WIFI_ECN_WPA2_PSK
+#define WIFI_SECURITY_TYPE                   WIFI_ECN_WPA2_PSK
 
-#define democonfigADU_DEVICE_MANUFACTURER "ST"
-#define democonfigADU_DEVICE_MODEL        "STM32L475"
-#define democonfigADU_UPDATE_PROVIDER     "ST"
-#define democonfigADU_UPDATE_NAME         "STM32L475"
-#define democonfigADU_UPDATE_VERSION      "1.0"
+#ifdef democonfigENABLE_DPS_SAMPLE
+    #define democonfigCHUNK_DOWNLOAD_SIZE    256
+#else
+    #define democonfigCHUNK_DOWNLOAD_SIZE    1024
+#endif /* democonfigENABLE_DPS_SAMPLE */
+
+#define democonfigADU_DEVICE_MANUFACTURER    "STMicroelectronics"
```

Comment:
nit: this will be annoying to type out fully for the script. Unless eric said spell it out I don't mind the short version or `STMicro`